### PR TITLE
Pic 2972 edit comment

### PR DESCRIPTION
**Pull request by rschallapalli**

### CODE REVIEW

File: integration-tests/integration/features/case-summary-comments/case-summary-comments.spec.js [hunk 1]
```diff
@@ -35,20 +35,67 @@ And('I should see the following comments with the comment, author and date comme
...
+        }
       })
     })
   })
 })
 
-And('I click Delete on the below comment located in table row {int}', ($int, $data) => {
+const getCommentsRow = rowNum => cy.get('.govuk-table__body > .govuk-table__row').eq(rowNum - 1)
+
+And('I click {string} on the below comment located in table row {int}', ($link, $int, $data) => {
```
Comment (johnyfarrar) [inline integration-tests/integration/features/case-summary-comments/case-summary-comments.spec.js:55]: Really like making these more generic - good work :)

File: server/routes/handlers/getUpdateCommentRequestHandler.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const getUpdateCaseCommentHandler = caseService => async (req, res) => {
+  const { params: { courtCode, hearingId, defendantId }, body: { caseId, comment, commentId } } = req
+
+  await caseService.updateCaseComment(caseId, commentId, comment, res.locals.user.name)
```
Comment (johnyfarrar) [inline server/routes/handlers/getUpdateCommentRequestHandler.js:4]: Should there be a try catch / with an error state? Question not a requirement

**Review by johnyfarrar — COMMENTED** [review]
A couple of comments - but I think it might be best to get this in? @paddybasi can you have a look too please?

File: server/routes/handlers/getUpdateCommentRequestHandler.js (hunk 2 above)
Comment (rschallapalli) [reply server/routes/handlers/getUpdateCommentRequestHandler.js:4]: I have suggested something on that lines to handle any unexpected errors and show some graceful error message to the user on this thread but doing so is not inline with the recommendation. Please see this thread https://mojdt.slack.com/archives/C01FR4HKS3A/p1686821180541949

So the catch block has no clue what to do with the caught error.

Comment (paddybasi) [reply server/routes/handlers/getUpdateCommentRequestHandler.js:4]: yeah think there is a generic error handler that will catch any unknowns, but yes I think specific error handling could also be a thing to do

**Review by paddybasi — APPROVED** [review]
LGTM

File: server/routes/handlers/getUpdateCommentRequestHandler.js (hunk 2 above)
Comment (johnyfarrar) [reply server/routes/handlers/getUpdateCommentRequestHandler.js:4]: Ok - I'll speak with the team about adding some work in the future about this.!

**Review by johnyfarrar — APPROVED** [review]

**Pull request merged**